extensions/amp-form/0.1/amp-form.js

```diff
@@ -274,8 +274,13 @@ export class AmpForm {
   renderTemplate_(data = {}) {
     const container = this.form_.querySelector(`[${this.state_}]`);
     if (container) {
+      const randId = `rendered-message-${Math.random() * 1000}`;
```

Comment:
I am happy to do this. Another option would be just to use a `counter` when initializing every `AmpForm` instance similar to what we do in `resources` when initializing new resources.